src/release/breaking-changes/multi-touch-scrolling.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Default multitouch scrolling
+title: Default multi-touch scrolling
```

Comment:
Actually, I checked this out very recently. "Multitouch" is becoming more accepted, so I went with no hyphen.

Reply:
Looks weird to me, but I can learn to accept it :P - Reverted this change.